setup.py

```diff
@@ -31,6 +31,7 @@ def run_meson_build():
     sysargs = meson_call.split(" ")
     sysargs = [arg for arg in sysargs if arg != ""]
     p1 = subprocess.run(sysargs, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    print(p1.stdout.decode())
```

Comment:
For debugging builds, it is incredibly useful to see the meson configure output even when it succeeds, as it lists the compilers and linkers that were found.

I'm not married to this exactly implementation, but I think it should be _possible_ to easily enable this output without having to patch setup.py.

Reply:
Yeah this is fine with me and is something I have been meaning to do., We do the same for the build step (but currently only if it fails).

Reply:
Actually, is the `print(p2.stdout)` duplicate with line 52? Can remove the latter if we want to always print build logs.

Reply:
Fixed